website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -62,26 +110,29 @@ company-name:
       host: hostname.region.redshift.amazonaws.com
       user: username
       password: password1
-      port: 5439
       dbname: analytics
       schema: analytics
+
+      # Optional Redshift configs:
+      port: 5439
```

Comment:
95% certain that this is a required parameter that has no default so it must be specified. could be a nice low-hanging UX improvement thought to make `5439` the default!

[source](https://github.com/dbt-labs/dbt-redshift/blob/919b39140412ff4ec616832ebc59aff4258139da/dbt/adapters/redshift/connections.py)

Reply:
before merging, @jiezhen-chen is port a required parameter?

Reply:
Yes, it is! The docs should be updated with the accurate information on `port`